### ISSUE TITLE
Should we add a description column to the tables?

### ISSUE THREAD

**Issue by starikcetin** [starikcetin/awesome-upm#40]
I am thinking about adding a description column in between the name and URL columns. It will be good for a brief summary of what the package does, but will also make the table more crowded.

What do you think?

**Comment by favoyang** [comment]
How about:

| Name | URL |
| -- | -- |
| ClassImpl | https://github.com/pipe01/classimpl |

=> 

| Title | Description |
| -- | -- |
| [ClassImpl](https://github.com/pipe01/classimpl) | Like a faking library, but oriented towards dynamic modification at runtime. |

**Comment by JimmyCushnie** [comment]
I am strongly in favor of @favoyang 's proposal.

@starikcetin , I can do this if you're pressed for time right now.

**Comment by starikcetin** [comment]
I like @favoyang 's proposal as well. Let it be our new format, then.
Thank you @JimmyCushnie , please feel free to do so.

**Comment by JimmyCushnie** [comment]
OK, I've finished doing this. I did my best to create a markdown format that keeps things nicely lined up, which has made the source file a whopping 254 characters wide.

Many of the descriptions are editorialized by yours truly. I tried to sum up the package's function more nicely than the official tagline.

I also took the liberty of removing Cinemachine, as that's now an official Unity package.
